.github/workflows/create_release.yml

```diff
@@ -0,0 +1,176 @@
+name: Create Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      release_type:
+        type: choice
+        description: Select release type
+        options:
+          - patch
+          - minor
+          - major
+
+env:
+  STENCIL_VERSION_FILE: stencil-library/package.json
+  VERSION_EXTRACT_PATTERN: '"version": "([^"]+)",'
+
+jobs:
+  create_release:
+    name: Create Release 
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Setup git credentials
+        run: |
+          git config user.name 'Act as JustiFi Machina'
+          git config user.email 'justifi.machina@justifi.ai'
+          git config user.password ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Generate versions
+        uses: HardNorth/github-version-generate@v1.2.0
+        with:
+          version-source: file
+          version-file: ${{ env.STENCIL_VERSION_FILE }}
+          version-file-extraction-pattern: ${{ env.VERSION_EXTRACT_PATTERN }}
+
+      # Compile & Link Stencil
+      - name: Install stencil-library
+        run: yarn --cwd stencil-library/ install
```

Comment:
Oh nice, didn't know about this `--cwd` flag. That's nice for just working in this repo 😄